README.md

```diff
@@ -1,2 +1,39 @@
-# operator-iq
-IBM Operator for Nexus IQ
+# Sonatype Nexus IQ Certified Operator
+Red Hat certified OpenShift Operator for installing Sonatype Nexus IQ Server
+to an OpenShift cluster.
+
+## Building from Source for Local Development and Testing
+
+To develop and test locally, you'll use CodeReady Containers on your workstation
+and push your operator image to quay.io to make it available for installation.
+
+1. Install [https://developers.redhat.com/products/codeready-containers/overview](CodeReady Containers)
```

Comment:
Little syntax error: the link goes in `()` and the text in the `[]`.

Reply:
I flubbed that again!